test/modules/overlays.js

```diff
@@ -263,23 +263,23 @@
             var zoom = viewport.viewportToImageZoom(viewport.getZoom(true));
             var expectedWidth = 124 * zoom;
             var expectedHeight = 132 * zoom;
-            Util.assessNumericValue(assert, $("#overlay").width(), expectedWidth, epsilon,
+            Util.assessNumericValue(assert, $("#overlay-wrapper-overlay").width(), expectedWidth, epsilon,
```

Comment:
Would it make more sense to make sure the passed in element (i.e. just `#overlay`) is the width and height we expect? We don't want the wrapping process to change this.

Reply:
I've checked and we can absolutely use size.  I've changed this.